styles/styles.css

```
@@ -0,0 +1,696 @@
+  padding-top: 64px;
+  text-align: center;
+}
+
+.recommend-title {
+  margin: 0 0 20px 0;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.img-icon {
+  margin-bottom: 30px;
+}
+
+.recommend-subtitle {
+  margin-top: 0;
+  margin-bottom: 30px;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.recommend-desc {
+  margin-top: 0;
+  margin-bottom: 64px;
+  font-size: 18px;
+  text-align: center;
+}
+
+.recommend-text {
+  margin: 0;
+  font-size: 18px;
+  text-align: center;
+}
+
+.recommend-list {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 400px 400px 400px;
+}
+
+.recommend-item {
+  list-style-type: none;
+  background: #ffffff;
+}
+
+.recommend-item-wrap {
+  padding: 80px 34px;
+  text-align: center;
+}
+
+.recommend-item-active {
+  background: rgba(131, 179, 211, 0.12);
+}
+
+/* Hotel search */
+.section-hotel-search {
+  background-color: #ffffff;
+  text-align: center;
+}
+
+.hotel-wrap {
+  width: 592px;
+  margin: 0 auto;
+  padding: 96px 0;
+  padding-top: 96px;
+  padding-bottom: 104px;
+}
+
+.hotel-search-title {
+  margin-top: 0;
+  margin-bottom: 20px;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.hotel-search-desc {
+  margin-top: 0;
+  margin-bottom: 54px;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+/* Subscribe */
+.section-subscribe  {
+   padding: 96px 0;
+   text-align: center;
+}
+
+.section-subscribe-images {
+  background-color: #e4f1f9;
+  background-image: url("../images/subscribe/subscribe-bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.subscribe-container {
+  width: 544px;
+  margin: 0 auto;
+}
+
+.section-subscribe-images .subscribe-title,
+.section-subscribe-images .subscribe-desc {
+  color: #ffffff;
+}
+
+.subscribe-title {
+  margin-top: 0;
+  margin-bottom: 20px;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+  color: #000000;
+
+}
+
+.subscribe-desc {
+  margin-top: 0;
+  margin-bottom: 54px;
+  font-size: 22px;
+  line-height: 26px;
+  text-align: center;
+  color: #000000;
+}
+
+/* Footer */
+.footer-page  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 45px 0;
+}
+
+.contacts-phone-link {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+  text-decoration: none;
+}
+
+/* Social list */
+.social-list{
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.social-list-item {
+  list-style-type: none;
+}
+
+.social-list-vk {
+  width: 24px;
+  height: 14px;
+  display: block;
+  mask: url("../images/svg/social/vk.svg") no-repeat center;
+  -webkit-mask: url("../images/svg/social/vk.svg") no-repeat center;
```

```suggestion
  -webkit-mask: url("../images/svg/social/vk.svg") no-repeat center;
  mask: url("../images/svg/social/vk.svg") no-repeat center;
```

Всегда сначала пишется свойство с префиксом, потом обычное. 
Чтобы когда свойство попадет в стандарт, отображалось именно оно, а не так как было в его тестовой версии.